WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -3565,17 +3567,18 @@ e_byzantium = {
 		capital = 560 # Cherson
 		
 		hellenic_pagan = 600
-		bosporan_faith = 200
+		bosporan_faith = 1000
 		
 		allow = {
-			hidden_tooltip = {
+			OR = {
+				religion = bosporan_faith
 				OR = {
-					ai = no
 					AND = {
 						religion_group = christian
 						OR = { 
 							culture_group = byzantine
 							culture_group = latin
+							has_romance_culture_trigger = yes
```

Comment:
It is a DE JURE kingdom of the Byzantine Empire. There shouldn't BE limits on it, but since there are, it should apply the same rules as the rest of the ERE.